.github/workflows/release.yml

```diff
@@ -25,17 +25,48 @@ jobs:
           key: ${{ runner.os }}-go-${{ hashFiles('go.sum') }}
           restore-keys: |
             ${{ runner.os }}-go
+      - name: cache cargo
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            tools/optimizer-server/target/
+          key: ${{ runner.os }}-cargo-${{ hashFiles('tools/optimizer-server/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-cargo
       - name: build nydus-snapshotter
         run: |
           go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.51.2
           export PATH=$PATH:$(go env GOPATH)/bin
           make
+      - name: build optimizer
+        run: |
+          rustup component add rustfmt clippy
+          make optimizer
```

Comment:
It's better to release static binaries of optimizer

Reply:
Got it.
I also add a package `nydus-snapshotter-static-vX.X.X-x86_64.tgz`, which contains all static binaries. So, can we keep binaries in `nydus-snapshotter-vX.X.X-x86_64.tgz` is not static?

Reply:
In my machine, I got an error:
```
nydus-snapshotter/optimizer-server: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by nydus-snapshotter/optimizer-server)
nydus-snapshotter/optimizer-server: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by nydus-snapshotter/optimizer-server)
nydus-snapshotter/optimizer-server: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by nydus-snapshotter/optimizer-server)
```

It is better to release static binary for optimizer-server.